pages/docs/integrations/grouparoo-calculated-property.mdx

```diff
@@ -1,24 +1,94 @@
 ---
-title: "@grouparoo / calculated-property"
-date: "2021-10-12"
+title: "Calculated Property"
+date: "2022-1-18"
 pullQuote: "Learn how to work with the Calculated Property Plugin as a Source to generate custom transformed properties."
 pageNavSelector: "h2"
 ---
 
-<PluginDocsLinks plugin="CalculatedProperty" />
+NPM: [@grouparoo/calculated-property](https://www.npmjs.com/package/@grouparoo/calculated-property)
```

Comment:
Were these links wrong?  This `<PluginDocsLinks />` component is used across all of our Plugins.  If it's generating the incorrect links we should update it!

Reply:
I'll look into it, but somehow it was omitting the hyphen from the name and the links were invalid.

Reply:
Figured this out. It was just a matter of how the variable was declared. It works now.